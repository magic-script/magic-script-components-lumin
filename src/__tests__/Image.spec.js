//
import React from "react";
import { create } from "react-test-renderer";
import { Image } from "magic-script-components";

describe("Image component", () => {
  test("Matches the snapshot", () => {
    const image = create(
      <Image filePath="test_file.png" width={0.2} height={0.2} />
    );
    expect(image.toJSON()).toMatchSnapshot();
  });
});
