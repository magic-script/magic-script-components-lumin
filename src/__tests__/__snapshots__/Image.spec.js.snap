// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component Matches the snapshot 1`] = `
<image
  filePath="test_file.png"
  height={0.2}
  width={0.2}
/>
`;
